### PR TITLE
docs(cn): translate src/content/loaders/exports-loader.md

### PR DESCRIPTION
**Pull request by jacob-lcs**
_describe your changes..._

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.
- [x] Do not abandon your Pull Request: [Stale Pull Requests][3].
- __Remove these instructions from your PR as they are for your eyes only.__

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/
[3]: https://webpack.js.org/contribute/#pull-requests

### CODE REVIEW

**Comment by jacob-lcs** [comment]
@QC-L

File: src/content/loaders/exports-loader.md [hunk 1]
```diff
@@ -16,36 +16,36 @@ repo: https://github.com/webpack-contrib/exports-loader
...
+允许对源文件设置 `module.exports`/`export`。
 
-Useful when a source file does not contain exports or something does not export.
+当源文件不包含 exports 或者有一些内容没有 export 时是有用的。
 
-For further hints on compatibility issues, check out [Shimming](/guides/shimming/) of the official docs.
+想要获取有关兼容性的进一步提示，请查看官方文档中的 [Shimming](/guides/shimming/)。
 
-> ⚠ By default loader generate ES module named syntax.
+> ⚠ 默认情况下，loader 生成命名为 syntax 的 ES module。
```
Comment (QC-L) [inline src/content/loaders/exports-loader.md:25]: 这里理解有问题吧？
```suggestion
> ⚠ 默认情况下，loader 生成具名的 ES module。
```

Comment (QC-L) [inline src/content/loaders/exports-loader.md]:
```suggestion
> ⚠ 默认情况下，loader 生成具名的 ES module。
```

Comment (QC-L) [reply src/content/loaders/exports-loader.md]: 感觉这里就是句法的意思？被命名的句法/语法 -> 具名

Comment (jacob-lcs) [reply src/content/loaders/exports-loader.md]: 确实有点问题，我修改下

Comment (jacob-lcs) [reply src/content/loaders/exports-loader.md]: > 感觉这里就是句法的意思？被命名的句法/语法 -> 具名

已修改

File: src/content/loaders/exports-loader.md [hunk 2]
```diff
@@ -241,45 +241,45 @@ export { Foo };
...
+- `[named [name] [name]Alias]` - 生成名为 exports 的 ES 模块，并导出等于另一个名称下的文件名的值., 对于 `single.js` 将会是 `single` and 和`singleAlias`, 生成 `export { single as singleAlias };`.
 
-> ⚠ You need to set `type: "commonjs"` to use `single` or `multiple` syntaxes.
+> ⚠ 你需要设置 `type: "commonjs"` 才能使用 `single` 或者 `multiple` 语法。
 
-> ⚠ Aliases can't be used together with `default` or `single` syntaxes.
+> ⚠ 别名不能与 `default` 或者 `single` 语法一起使用。
 
-##### Examples {#examples}
+##### 例子 {#examples}
```
Comment (QC-L) [inline src/content/loaders/exports-loader.md]:
```suggestion
##### 示例 {#examples}
```

Comment (jacob-lcs) [reply src/content/loaders/exports-loader.md]: done

**Review by QC-L — APPROVED** [review]

**Pull request merged**